test/async-hooks/test-graph.tcp.js

```diff
@@ -17,7 +17,7 @@ const server = net
 
 server.listen(common.PORT);
 
-net.connect({ port: server.address().port, host: server.address().address },
+net.connect({ port: server.address().port, host: '::1' },
```

Comment:
I'd rather have this as a property of `common` like:
https://github.com/nodejs/node/blob/73894dad9d74671377c79dc7542b41813237c3cc/test/common/index.js#L200-L221